qemu/tests/hugepage_reset.py

```diff
@@ -107,9 +125,9 @@ def heavyload_install():
                 "No node on your host has sufficient free memory for " "this test."
             )
     hp_config = test_setup.HugePageConfig(params)
+    if params.get("on_numa_node"):
+        allocate_largepages_per_node()
```

Comment:
@mcasquer , code LGTM, I just want to confirm with you that if the node mem is not enough, still setup or better to raise error or skip the test.

Reply:
Mmmm @PaulYuuu good point, I think that situation should be handled, perhaps with a try block, I'll send an update of this

Reply:
@PaulYuuu added a try block that will cancel the case ig there's no enough memory, faked example:
```
 (2/2) Host_RHEL.m9.u6.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.6.0.x86_64.io-github-autotest-qemu.hugepage_reset.on_numa_node.q35: STARTED
 (2/2) Host_RHEL.m9.u6.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.6.0.x86_64.io-github-autotest-qemu.hugepage_reset.on_numa_node.q35: CANCEL: 18 (expecting 400) hugepages is set on the node 0, please check if the node has enough memory (12.51 s)
```